user_profile.py

```diff
@@ -0,0 +1,172 @@
+from nose.tools import set_trace
+import json
+from problem_details import *
+from flask.ext.babel import lazy_gettext as _
+
+class ProfileController(object):
+    """Implement the User Profile Management Protocol.
+
+    https://github.com/NYPL-Simplified/Simplified/wiki/User-Profile-Management-Protocol
+    """
+
+    MEDIA_TYPE = "vnd.librarysimplified/user-profile+json"
+    LINK_RELATION = "http://librarysimplified.org/terms/rel/user-profile"
+    SETTINGS_KEY = 'settings'
+    
+    def __init__(self, store):
+        """Constructor.
+
+        :param store: An instance of ProfileStore.
+        """
+        self.store = store
```

Comment:
It represents the user themselves. In my follow-up branch it's a wrapper around the `Patron` object.